practice-one/src/styles/components/food-card.css

```diff
@@ -0,0 +1,62 @@
+.card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 310px;
+  height: 516px;
+  border-radius: 25px;
+  background: var(--color-gradient-strong);
+  position: relative;
+}
+
+.card-filter {
+  position: relative;
+  background: var(--color-gradient-warm);
+  width: 253.93px;
+  height: 253.93px;
+  border-radius: 50%;
+}
+
+.food-price {
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  border-radius: 50%;
+  border: 5px solid var(--color-light);
+  background-color: var(--color-accent);
+  text-align: center;
+  color: var(--color-light);
+  font-size: 18px;
+  top: 175px;
+  left: 175px;
+}
+
+.food-image {
+  position: absolute;
+  width: 198px;
+  height: 198px;
+  border: 10px solid var(--color-light);
+  background-color: var(--color-accent);
+  border-radius: 50%;
+  top: 20px;
+  left: 20px;
+}
+
+.food-card-header {
+  color: var(--color-primary);
+  font-family: var(--font-family-bold);
+  font-size: 24px;
+}
+
+.food-card-text {
+  color: var(--color-dark);
+  text-align: center;
+  position: absolute;
+  margin: 20px;
+  top: 350px;
+}
+
+.card-button {
+  position: absolute;
+  top: 500px;
+}
```

Comment:
Please recheck the position of this button, look at your image attach in description, seem like it was wrong

Reply:
i checked and fixed it